saleor/account/models.py

```diff
@@ -160,12 +161,17 @@ class User(PermissionsMixin, ModelWithMetadata, AbstractBaseUser):
 
     objects = UserManager()
 
-    class Meta(ModelWithMetadata.Meta):
+    class Meta:
         ordering = ("email",)
         permissions = (
             (AccountPermissions.MANAGE_USERS.codename, "Manage customers."),
             (AccountPermissions.MANAGE_STAFF.codename, "Manage staff."),
         )
+        indexes = [
+            # Orders searching index
+            GinIndex(fields=["email", "first_name", "last_name"]),
+        ]
+        indexes.extend(ModelWithMetadata.Meta.indexes)
```

Comment:
we usually keep stuff in meta as single objects, maybe that would be more consistent?
```
indexes = [
    *ModelWithMetadata.Meta.indexes,
    # Orders searching index
    GinIndex(fields=["email", "first_name", "last_name"]),
]
```